impeller/renderer/capabilities.h

```diff
@@ -15,32 +15,95 @@ class Capabilities {
  public:
   virtual ~Capabilities();
 
+  /// @brief  Whether the context backend supports attaching offscreen MSAA
+  ///         color/stencil textures.
   virtual bool SupportsOffscreenMSAA() const = 0;
 
+  /// @brief  Whether the context backend supports binding Shader Storage Buffer
+  ///         Objects (SSBOs) to pipelines.
   virtual bool SupportsSSBO() const = 0;
 
+  /// @brief  Whether the context backend supports blitting from a given
+  ///         `DeviceBuffer` view to a texture region (via the relevant
+  ///         `BlitPass::AddCopy` overloads).
   virtual bool SupportsBufferToTextureBlits() const = 0;
 
+  /// @brief  Whether the context backend supports blitting from one texture
+  ///         region to another texture region (via the relevant
+  ///         `BlitPass::AddCopy` overloads).
   virtual bool SupportsTextureToTextureBlits() const = 0;
 
+  /// @brief  Whether the context backend is able to support pipelines with
+  ///         shaders that read from the framebuffer (i.e. pixels that have been
+  ///         written by previous draw calls in the current render pass).
+  ///
+  ///         Example of reading from the first color attachment in a GLSL
+  ///         shader:
+  ///         ```
+  ///         layout(set = 0, binding = 0, input_attachment_index = 0)
```

Comment:
Nit: You can skip the layout attributes as the compiler will autogenerate them. We don't usually need or set them in Impeller.

Reply:
Removed.